aws/ami/build_deb_ami.sh

```diff
@@ -123,8 +123,11 @@ SSH_USERNAME=ubuntu
 
 if uname -m | grep x86_64 ; then
   INSTANCE_TYPE="c4.xlarge"
-else
+elif uname -m | grep aarch64 ; then
   INSTANCE_TYPE="a1.xlarge"
+else
+  echo "Unsupported architecture"
```

Comment:
We should print out the architecture here.

You could simplify the whole thing to:

You could simplify this to:

```bash
arch="$(uname -m)"
case "$arch" in
  "x86_64")
    INSTANCE_TYPE="c4.xlarge"
    ;;
  "aarch64")
    INSTANCE_TYPE="a1.xlarge"
    ;;
  *)
    echo "Unsupported architecture: $arch"
    exit 1
esac
```